frontend/src/components/MailingForm.vue

```diff
@@ -31,23 +35,21 @@ import APIClient from '../utils/APIClient';
 import { MAILING_URL } from '../utils/Constants';
 
 export default {
-  name: 'EnquiryForm',
+  name: 'MailingForm',
   props: ['type'],
   data: () => ({
     valid: true,
     name: '',
-    nameRules: [
-      (v) => !!v || 'Name is required',
-    ],
     email: '',
-    emailRules: [
-      (v) => !!v || 'E-mail is required',
-      (v) => /.+@.+\..+/.test(v) || 'E-mail must be valid',
-    ],
     body: '',
-    bodyRules: [
-      (v) => !!v || 'Message is required',
-    ],
+    rules: {
+      required: (value) => !!value || 'Required',
+      email: (value) => {
+        // eslint-disable-next-line max-len
+        const pattern = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
_nit_: I remember using a third party JS library to validate email. "Yup". I think it was called.

Reply:
"Yup" seems like a very complete validation package. But it is a bit overkill for just email validation. What do you think about using this [email-validator](https://www.npmjs.com/package/email-validator) instead?